sections/user/edit.php

```diff
@@ -1122,6 +1134,30 @@ function checked($Checked) {
                             </div>
                         </td>
                     </tr>
+
+                </table>
+
+                <table class="Form-rowList" variant="header" id="token_settings">
```

Comment:
Recommend: `api token settings` should belong to `access settings`

Reply:
I've updated this. The reason why i chose to keep it separate is because the backend has the capability to support multiple api keys for a user. I've bypassed it to get things done quickly.

Down the road should you guys decide to add support for multiple api key, it can be enabled.